### PR TITLE
[RISCV] Lower fixed-length strided VP loads and stores for zvfhmin/zvfbfmin

### PR DESCRIPTION
**Pull request by lukel97**
Similarly to #114731, these don't actually require any instructions from the extensions.

The motivation for this and #114731 is to eventually enable isLegalElementTypeForRVV for f16 with zvfhmin and bf16 with zvfbfmin in order to enable scalable vectorization.

Although the scalable codegen support for f16 and bf16 is now complete enough for anything the loop vectorizer may emit, enabling isLegalElementTypeForRVV would make certian hooks like isLegalInterleavedAccessType and isLegalStridedLoadStore return true for f16 and bf16. This means SLP would start emitting these intrinsics, so we need to add fixed-length codegen support.

### CODE REVIEW

**Review by wangpc-pp — APPROVED** [review]
LGTM.

File: llvm/lib/Target/RISCV/RISCVISelLowering.cpp [hunk 1]
```diff
@@ -1347,6 +1347,10 @@ RISCVTargetLowering::RISCVTargetLowering(const TargetMachine &TM,
         setOperationAction({ISD::STRICT_FP_ROUND, ISD::STRICT_FP_EXTEND}, VT,
                            Custom);
 
+        setOperationAction({ISD::EXPERIMENTAL_VP_STRIDED_LOAD,
```
Comment (topperc) [inline llvm/lib/Target/RISCV/RISCVISelLowering.cpp:1350]: There's a FIXME on line 1342 that should be changed.

Comment (lukel97) [reply llvm/lib/Target/RISCV/RISCVISelLowering.cpp:1350]: Yeah, I hoping to be able to add lowering for the rest of the nodes mentioned there as well too. Will update

Comment (topperc) [reply llvm/lib/Target/RISCV/RISCVISelLowering.cpp:1350]: I went ahead and merged the setOperationActions and updated the FIXME in 8d023b7d660dc5347740cb5e17e1dfbf8ff93686

Comment (lukel97) [reply llvm/lib/Target/RISCV/RISCVISelLowering.cpp:1350]: Thanks

**Pull request merged**